lua/rocks/commands.lua

```diff
@@ -119,8 +122,10 @@ end
 ---@type { [string]: RocksCmd }
 local rocks_command_tbl = {
     update = {
-        impl = function(_)
-            require("rocks.operations").update()
+        impl = function(_, opts)
+            require("rocks.operations").update(nil, {
+                skip_prompt = opts.bang,
```

Comment:
bikeshedding: skip_prompt -> force_major_updates ? or skip_semver_prompt.

Reply:
That's a typo. I renamed it to `skip_prompts` (plural), because there could be more than one prompt, as is the case with `install`.